src/main/scala/com/kinja/amqp/persistence/InMemoryMessageBuffer.scala

```diff
@@ -95,11 +95,14 @@ class InMemoryMessageBuffer extends Actor with ActorLogging {
 			val age = time - message.createdTime.getTime
 			Math.max(acc, age)
 		}
-
-		logger.info(s"There are ${messageBuffer.size} message in the buffer, " +
-			s"with average age of $averageAge millis, " +
-			s"max age is $maxAge millis")
-		logger.info(s"There are ${confirmations.size} confirmations in the buffer, and they are $confirmations")
+		if (messageBuffer.size > 0) {
```

Comment:
I think `nonEmpty` would be a better performing solution here and a few lines later as well

Reply:
Fixed.